genesis/sensors/base_sensor.py

```diff
@@ -162,31 +174,104 @@ def read_ground_truth(self, envs_idx: List[int] | None = None):
         """
         return self._get_formatted_data(self._manager.get_cloned_from_cache(self, is_ground_truth=True), envs_idx)
 
+    def add_recorder(self, handler: "DataHandler", rec_options: RecordingOptions | None = None):
+        """
+        Add a sensor data recorder that processes data from this sensor.
```

Comment:
This is not sufficient as documentation. This method is part of the public API. What it does should be very clear. Besides you are not explaining what are the parameters. In this case, these parameters are instances of very specific objects, so you should even explain how these objects can be instantiated in a classical workflow, eventually with an example code snippet and cross-reference to other documentation if needed.

I would expect at the very least 8 lines of documentation for this method.